### PR TITLE
Update default Monero node URLs and info

### PR DESCRIPTION
**Pull request by da-kami**
xmr.to nodes are not served at the xmr.to domain but moved to melo.tools

### CODE REVIEW

**Review by thomaseizinger — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Fixed
+
+- An issue where the default Monero node connection string would not work, because the public nodes were moved to a different domain.
+  The default monerod nodes were updated to use the [melo tool nodes](https://melo.tools/nodes.html).
```
Comment (thomaseizinger) [inline CHANGELOG.md:13]: Now that we deleted the old release, this should be part of the old changelog.

**Review by thomaseizinger — CHANGES_REQUESTED** [review]

**Review by rishflab — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (da-kami) [reply CHANGELOG.md]: rebased on https://github.com/comit-network/xmr-btc-swap/tree/revert-release-0.8.0

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.8.0] - 2021-07-07
```
Comment (thomaseizinger) [inline CHANGELOG.md:10]: This should not be here no?

**Review by thomaseizinger — CHANGES_REQUESTED** [review]

**Pull request merged**